src/Nordea/Components/Tooltip.elm

```diff
@@ -123,46 +136,123 @@ withVisibility visibility (Tooltip config) =
 view : List (Attribute msg) -> List (Html msg) -> Tooltip msg -> Html msg
 view attrs children (Tooltip config) =
     let
+        arrowDistanceFromCornerInPct =
+            15
```

Comment:
Is the percentage relative to the width/height of the tooltip? If that is the case is it better to make it static?

Reply:
Yes, 15% is calculated depending on the width/height of the content. The way it works right now is arrow stays central always to the content being tooltipped, but the positioning of the box changes.
By static, if you mean dropping the pct and doing it with rem, It might look a bit odd for wide or tall boxes. It is more robust to height/width changes this way I think.

Reply:
The need for these extra arrow placements is to be able to keep the tooltip on-screen. So my concern was whether the relative positioning may push parts of the tooltip outside the screen as the size increases. For example for our coachmark case, double the width and I guess the right side may be outside the screen? There may be a higher risk for mobile views where we have much less padding and the icon appears closer to the edge.

Not sure if a realistic concern tho, have not tested 😄

Reply:
you are right it might break in that way and go out of screen for large input. might make sense to make it static.

It is especially tricky to make the horizontal one work because the height is so short there. I will have another look if it generalizes good.
![image](https://github.com/SGFinansAS/elm-components/assets/34012758/44ec1fd4-bdf8-47ff-95d9-1aefa7a51934)